docs/node-groups.md

```diff
@@ -166,3 +166,36 @@ The example below demonstrates how you can customize a Fargate profile for your
       }
     }
 ```
+
+### Windows Self-Managed Node Groups
+
+The example below demonstrates the minimum configuration required to deploy a Self-managed node group of Windows nodes. Refer to the [AWS EKS user guide](https://docs.aws.amazon.com/eks/latest/userguide/windows-support.html) for more information about Windows support in EKS.
+
+```hcl
+  # SELF-MANAGED NODE GROUP with Windows support
+  enable_windows_support = true
+
+  self_managed_node_groups = {
```

Comment:
This is nice! It would be very useful if we provide example with **all the options**(e.g., user data, labels, taints etc.). Docs should provide full details on how to consume Windows node groups so users doesn't need to goto multiple places to figure out how to use the module

Reply:
AFAIK Windows self-managed node groups support all the options as those supported by Linux self-managed NGs. At this time I have added the minimum / typically needed options. We can expand this section after rigorous testing of each option and various combinations of options.